website/src/pages/home/_UseCases.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FunctionComponent, useLayoutEffect, useRef, useState } from 'react'
+import { Link } from 'gatsby'
+
+import { MdiReactIconComponentType } from 'mdi-react'
+import LockOutlineIcon from 'mdi-react/LockOutlineIcon'
+import ClockOutlineIcon from 'mdi-react/ClockOutlineIcon'
+import CheckboxMarkedCircleOutlineIcon from 'mdi-react/CheckboxMarkedCircleOutlineIcon'
+import CachedIcon from 'mdi-react/CachedIcon'
+import LaptopIcon from 'mdi-react/LaptopIcon'
+
+import { ContentSection } from '../../components/content/ContentSection'
+import { useWindowWidth } from '../../hooks'
+import { breakpoints } from '../../breakpoints'
+import { buttonStyle, buttonLocation } from '../../tracking'
+
+interface UseCases {
+    icon: MdiReactIconComponentType
+    title: string
+    description: string
+    link: {
+        ['href']: string
+        ['text']: string
+    }
+}
+
+const useCases: UseCases[] = [
+    {
+        icon: LockOutlineIcon,
+        title: 'Improve code security',
+        description: 'Find, fix, and track vulnerable code quickly across your entire codebase.',
+        link: {
+            href: '/use-cases/vulnerabilities',
+            text: 'Learn more about code security',
+        },
+    },
+    {
+        icon: ClockOutlineIcon,
+        title: 'Onboard developers',
+        description: 'Decrease time to first commit with codebase onboarding and knowledge sharing.',
+        link: {
+            href: '/use-cases/onboarding',
+            text: 'Learn more about onboarding',
+        },
+    },
+    {
+        icon: CheckboxMarkedCircleOutlineIcon,
+        title: 'Resolve incidents',
+        description: 'Identify the root cause in code and fix the issue everywhere, faster.',
+        link: {
+            href: '/use-cases#resolve-incidents-faster',
+            text: 'Learn more about incident response',
+        },
+    },
+    {
+        icon: CachedIcon,
+        title: 'Promote code reuse',
+        description: 'Find existing code for reuse and contribute to a more coherent codebase.',
+        link: {
+            href: '/use-cases#streamline-code-reuse',
+            text: 'Learn more about code reuse',
+        },
+    },
+    {
+        icon: LaptopIcon,
+        title: 'Boost code health',
+        description: 'Improve code health with large-scale changes, and track key initiatives.',
+        link: {
+            href: '/use-cases#boost-code-health',
+            text: 'Learn more about code health',
+        },
+    },
+]
+
+const UseCases: FunctionComponent = () => {
+    const box = useRef(null)
+    const [boxHeight, setBoxHeight] = useState<number>(0)
+
+    const windowWidth = useWindowWidth()
```

Comment:
This hook is awesome! My last project used Vuetify for our UI library, and it had a built in reactive [boolean variable](https://vuetifyjs.com/en/features/breakpoints/#breakpoint-conditionals) for this, to dynamically output the current breakpoint (ie `$vuetify.breakpoint.xs, `$vuetify.breakpoint.mdAndUp`). I'm surprised it doesn't seem like Bootstrap has this :/  CC: @bretthayes @zlonko

Reply:
Yeah, I don't think Bootstrap has anything that we could reference elegantly other than the SCSS which isn't helpful in this case.

With Tailwind CSS, we can actually import and reference the theme since it's just a js config, and then we can use the `useWindowWidth` hook since it's so lightweight. ✨ 

Tailwind 1... Bootstrap 0... lol 🙌🏻

Reply:
awesome! excited to make the jump to tailwind :D